package.js

```diff
@@ -1,14 +1,14 @@
 Package.describe({
   summary: 'Define and run db migrations.',
-  version: '1.1.0',
+  version: '2.0.0',
```

Comment:
it may break things but I agree on the changes

Reply:
I think this update is necessary as the internals will change significantly and there will be hard minimum Meteor requirement of 2.8.

Reply:
I think we can either release the full version or if you want @Grubba27 we can release a beta. Thoughts?